e2e/test.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: test
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test
+spec:
+  selector:
+    matchLabels:
+      app: test
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: test
+    spec:
+      serviceAccountName: test
+      containers:
+        - name: test
+          image: poke/test:latest
```

Comment:
`poke/test:latest`? maybe we should use `nginx:1.10` or something that is not `latest` and probably will change?